### ISSUE TITLE
(Some) new deployments are disappearing from the receiver locations map

### ISSUE THREAD

**Issue by j-sayers** [MotusDev/Motus-TO-DO#1194]
Location information is now stored at the station level instead of the deployment level. The [receiver deployment map](https://motus.org/data/receiversMap?lang=en) displays receiver deployments still, so many new deployments are not appearing on the map. But this doesn't happen in every case so in _some_ situations location information is being pulled from the stations table.

Two recent examples:

- A new deployment at an existing station which disappeared from the map: [Prince Edward Point BO](https://motus.org/data/project/stations?setStationId=11555)
- A brand new station that did appear on the map: [Atik](https://motus.org/data/project/stations?setStationId=12576)-1

**Comment by cjardine-bsc** [comment]
They have location in the base view. 

SELECT *
  FROM [motus].[dbo].[vwBase_receivers_deployments]
  where recvDeployID in ( 8618,8622)

**Comment by Hussell** [comment]
The single example given (PEPBO deployment 8618) doesn't have anything to do with latitude and longitude. The receiver map only displays receiver deployments which have isMobile = 0. The example is one of 34 receiver deployments in vwBase_receivers_deployments which have isMobile = NULL, compared to 5006 deployments with isMobile = 0 and 90 deployments with isMobile = 1.

There are only 14 such deployments in sandbox, which implies that something has changed recently so that isMobile = 0 isn't being set correctly on new deployments.

Digging into the views, isMobile comes from a join with lk_fixture_type. The 34 deployments with isMobile = NULL are all and only the deployments with fixture_type = NULL. When I tried creating a new deployment on sandbox, the page refused to save changes when I didn't enter a fixture type, so that probably isn't the issue. Though once a deployment has been created the fixture type can't be changed, which is user-unfriendly [what a euphemism].

This is where I've hit a dead end. I can't figure out how to create a deployment with a NULL fixture_type. @j-sayers do you have any idea how this can happen?

**Comment by j-sayers** [comment]
Yes fixture_type is required when manually entering the details. Looks like what's happening is
1) fixture_type is not copied over to a new deployment at an existing station. This may be because fixture_type is actually the only piece of information either not stored at the station level (location etc) or controlled by the check-box options when creating a new deployment (both receiver and antenna config are controlled here). So it just got missed.
2) the data validation only operates when the user is saving. Because a new deployment is already saved, the validation is just skipped. If you then activate edit mode for one of these deployments you will be forced to select fixture_type.

So we should at least:

- [ ] ensure that fixture_type is copied from the most recent deployment at a station to the newest deployment 
- [ ] update the receiver deployment fixture_types that are NULL to the value of the most recent deployment at those stations. I did this manually for a few in Ontario now and they are back on the map

**Comment by denislepage** [comment]
You can also let me know when you have an updated live branch with beta 11, i can look to deploy to the new tomcat.
